src/test/java/cucumber/steps/StepDefinitions.java

```diff
@@ -0,0 +1,45 @@
+package cucumber.steps;
+
+import controllers.DashboardController;
+import controllers.LaunchController;
+import io.cucumber.java.en.When;
+
+import java.util.ArrayList;
+
+import static helpers.UserTokenGenerator.getToken;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StepDefinitions {
+    private LaunchController launch = new LaunchController(Hooks.api);
```

Comment:
Why access ApiClient directrly?

Reply:
I've separated ApiClient for Hooks and for Steps. I think that now it is better